### PR TITLE
Resolve #743 - Add "author, updated ... ago and priority" in project page

### LINKED ISSUE #743: Add "author, updated ... ago and priority" in project page
Some useful information like "author" is not available in the project page.

Harry wrote in IRC:
> Could just have the same line "Created by http://tasks.hotosm.org/user/pedrito1414 - Updated about 2 hours ago - Priority: medium"  appear on the description

### PR DESCRIPTION
**Pull request by KaiDalgleish**
**This resolves #743**  
Updated the project description to include more information about the project, based off of similar code in "home.mako".

screen shot 2016-03-04 at 4 17 42 pm
- [x] Author username appears (there is no listed author for the demo project, so it doesn't show up in the screenshot)
- [x] Last update appears
- [x] Priority appears as text, not a number

(This is my first pull request to this project, so please feel free to make suggestions and corrections)

### CODE REVIEW

**Comment by pgiraud** [comment]
It looks very good!
Have you considered factorizing with `home.mako`? You copied some code from there, didn't you?
If this is too much work I can live with your change.

**Comment by KaiDalgleish** [comment]
Good idea; I'll work on that today.

**Comment by KaiDalgleish** [comment]
This was a little more complicated than I expected because in home.mako, the code was inside a function/def which is called for each project. Using `%include` to link to another template resulted in an AttributeError for any `project` reference.

I worked around that by making it into a separate `def` that could be imported and used anywhere. I put it in`custom

- [x] Made `display_project_info(project)` in `custom.mako`
    - Renders project status, author, update time, and priority with correct styling
- [x] Imported `display_project_info` into `home.mako`, `project.description.mako`


**Home View:**
screen shot 2016-03-05 at 8 07 45 pm

**Project View:**
screen shot 2016-03-05 at 8 08 01 pm

**Comment by KaiDalgleish** [comment]
Also, squashed commits :)

**Comment by pgiraud** [comment]
It looks excellent. Creating a `def` is definitely the way to go.
I have one last request. The `custom.mako` file is intended to be used for elements that should be (or least can be) overridden by application administrators. I don't think that the "project info" is something that is supposed to be changed. It can remain the same all over the different instances.

I see two options:
 - creating a dedicated `project.info.mako` that you can include using `namespace`,
 - creating a new `helpers.mako` templates in which we can put everything template partial that can be included in several places.

**Comment by pgiraud** [comment]
@KaiDalgleish Seen my last comment?

**Comment by KaiDalgleish** [comment]
@pgiraud Nope, hadn't see your comment! Thanks for the feedback. I wondered if that was the right place for it. 
To fix this I ...
- [x] Made `helpers.mako` for `display_project_info`
- [x] Changed references in `home.mako` and `project.description.mako`
- [x] squashed commits

File: osmtm/templates/home.mako [hunk 1]
```diff
@@ -112,7 +113,7 @@ sorts = [('priority', 'asc', _('High priority first')),
     from osmtm.mako_filters import markdown_filter
     if request.locale_name:
         project.locale = request.locale_name
-    priority = priorities[project.priority]
+    # priority = priorities[project.priority]
```
Comment (pgiraud) [inline osmtm/templates/home.mako]: Please don't leave any commented code. If it's not use just remove it.

File: osmtm/templates/project.description.mako [hunk 2]
```diff
@@ -16,13 +17,10 @@ from osmtm.mako_filters import markdown_filter
...
 </p>
 % endif
-% if project.private:
-<p class="text-muted">
-  <span class="glyphicon glyphicon-lock"></span>
-  ${_('Access to this project is limited')}
-</p>
-% endif
 <p>${project.description | markdown_filter, n}</p>
+<p id="info">
```
Comment (pgiraud) [inline osmtm/templates/project.description.mako]: Any reason why you add an id for this element?

**Comment by KaiDalgleish** [comment]
Thanks for feedback!
Cleaned up per @pgiraud 's comments:
- [x] Removed commented-out code
- [x] Removed unused `id`

**Comment by pgiraud** [comment]
This looks great.
However, I just figured out that your pull request has been done against 2.12. Is there any reason for that? I think the target is rather master since it's not a bug fix.
If this OK for your, can you please create a new pull request?

**Comment by KaiDalgleish** [comment]
Yeah, when I was first getting set up I had issues and was told to pull off of 2.12 instead of master. I'll submit a new request :)

**Pull request closed without merging**